static/preloadRenderer.js

```diff
@@ -7,9 +7,11 @@ const handleZoom = require('./preloads/handleZoom');
 handleErrors(document);
 handleZoom(document);
 
+// TODO discuss way to retrieve assets from 1 single point
```

Comment:
As far as I'm aware, these are the only 2 assets on the NEO blockchain currently, so this is a moot point for now.

Reply:
It was rather because there are several "ASSETS" declared throughout the client. It didn't seem very consistent. But I didn't think about the fact you mentioned above :P 
Either way, we should be able to expose the nOS scripthash (and other nep-5 token scripthashes) through the API to be able to do `const { NOS } = window.NOS.ASSETS; `